dashboard/app/controllers/home_controller.rb

```diff
@@ -35,22 +35,17 @@ def health_check
   GALLERY_PER_PAGE = 5
 
   def index
-    redirect_to '/courses'
+    redirect_to '/home'
   end
 
-  # Show /home for teachers.
   # Signed out: redirect to code.org
-  # Signed in teacher or have student_homepage cookie: render this page
-  # Signed in student: redirect to studio.code.org/courses
-
+  # Show /home for teachers and students
```

Comment:
nit: flip ordering to be consistent :)  So it's "Teacher or student: show /home"